Content.Server/Shuttles/Systems/ShuttleSystem.cs

```diff
@@ -120,6 +121,13 @@ private void OnShuttleStartup(EntityUid uid, ShuttleComponent component, Compone
             return;
         }
 
+        // Automatically anchor station grids
+        if (HasComp<BecomesStationComponent>(uid))
+        {
+            component.LinearDamping = 10000f; // tug with 10 thrusters can push this at 1 tile/sec
+            component.AngularDamping = 10000f;
+        }
+
```

Comment:
Wouldn't it be better to use the new ECS station prototype setup for this instead of hard-coding it in?

Reply:
I was not aware of this system. Would you mind linking me to the relevant PR?

Reply:
It's the one by moonheart #16570